From 2ae82a68a072c5692f4b2a9538d5d7595be70e51 Mon Sep 17 00:00:00 2001
From: Lee Jones <lee.jones@linaro.org>
Date: Wed, 12 Mar 2014 12:39:41 +0000
Subject: [PATCH] ahci: st: Only build for ST-Micro h/w

This device is designed specifically to run on ST-Microelectronics'
hardware. To ensure no attempts are made to run on anything incompatible
we add a dependency on ST architecture

Suggested-by: Bartlomiej Zolnierkiewicz <b.zolnierkie@samsung.com>
Signed-off-by: Lee Jones <lee.jones@linaro.org>
Signed-off-by: Tejun Heo <tj@kernel.org>
(cherry picked from commit 48c54df1305c4287cf8d8a344de6b70b1d56e234)
---
 drivers/ata/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/ata/Kconfig b/drivers/ata/Kconfig
index 3a41a55..445e594 100644
--- a/drivers/ata/Kconfig
+++ b/drivers/ata/Kconfig
@@ -100,6 +100,7 @@ config SATA_AHCI_PLATFORM
 config AHCI_ST
 	tristate "ST AHCI SATA support"
 	depends on SATA_AHCI_PLATFORM
+	depends on ARCH_STI
 	help
 	  This option enables support for ST AHCI SATA controller.
 
-- 
1.8.0

